Packs/RecordedFuture/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Recorded Future v2
+**Breaking Change** reputation commands **ip** **domain** **hash** and **url** changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
**Breaking Change** the reputation commands **ip**, **domain**, **hash**, and **url** were changed to return multiple entries (entry per indicator) instead of a single entry.
```